App/arca/components/sidebar.jsx

```diff
@@ -0,0 +1,171 @@
+    const imgBtn = document.querySelector('.img-btn');
+    const itemHelp = document.querySelector('.item-help');
+    const conHelp = document.querySelector('.con-help-center');
+    const imgArca = document.getElementById('arca-logo');
+
+    if (sidebar.clientWidth === 70) {
+      setTimeout(() => {
+        conDash.style.gridTemplateColumns = "260px 1fr";
+        sidebar.style.width = "260px";
+        btnSideBar.style.left = "245px";
+        imgBtn.src = "../assets/img/icons/angle-double-small-right-free-icon-font.svg";
+        imgArca.style.width = "85.5px";
+        imgArca.style.height = "35px";
+        conHelp.style.display = "grid";
+        itemHelp.style.display = "none";
+      }, 500);
+    } else {
+      setTimeout(() => {
+        conDash.style.gridTemplateColumns = "70px 1fr";
+        sidebar.style.width = "70px";
+        btnSideBar.style.left = "55px";
+        imgBtn.src = "../assets/img/icons/angle-double-small-left-free-icon-font.svg";
+        imgArca.style.width = "30px";
+        imgArca.style.height = "35px";
+        conHelp.style.display = "none";
+        itemHelp.style.display = "grid";
+      }, 500);
+    }
+  }
+
+  function menu() {
+    const sidebar = document.querySelector('.body-dashboard');
+    const btnSideBar = document.querySelector('.btn-des-sidebar');
+    const imgBtn = document.querySelector('.img-btn');
+    const imgArca = document.getElementById('arca-logo');
+    const btnMenu = document.querySelector('.con-sidebar-a');
+    const body = document.body;
+
+    sidebar.style.width = "260px";
+    conDash.style.gridTemplateColumns = "260px 1fr";
+    sidebar.style.width = "260px";
+    btnSideBar.style.left = "245px";
+    imgBtn.src = "../assets/img/icons/angle-double-small-right-free-icon-font.svg";
+    imgArca.style.width = "85.5px";
+    imgArca.style.height = "35px";
+
+    if (estadoMenu) {
+      sidebar.style.left = "-260px";
+      btnMenu.style.left = "-6000px";
+      btnMenu.style.right = "auto";
+      setEstadoMenu(false);
+      body.style.overflowY = "auto";
+    } else {
+      sidebar.style.left = "0px";
+      btnMenu.style.left = "auto";
+      btnMenu.style.right = "0px";
+      setEstadoMenu(true);
+      body.style.overflowY = "hidden";
+    }
+  }
+  return (
+    <>
+    <div className="head-sidebar">
+        <div className="menu" onClick={menu}>
+            <img src={menuimg} alt="Menu" />
+        </div>
+        <a className="arca" href="#">
+            <img src={arcalogo} alt="Arca logo" id="arca-logo" />
+        </a>
+    </div>
+<div className="body-dashboard">
+    <div className="con-item">
+        
+        <Link className="item-sidebar item-active" to="/Dashboard">
+            <div className="color-item color-item-active"></div>
+            <div className="con-ico-item">
+                <img src={dashboardimg} alt="Dashboard" />
+            </div>
+            <div className="name-item">Dashboard</div>
+        </Link>
```

Comment:
Create a nested component to define the ItemSidebar, the attributes could be 
"name", "url", "image"